modules/imaging_browser/help/imaging_browser.md

```diff
@@ -1,36 +1,17 @@
 # Imaging Browser
 
-The Imaging Browser allows users to browse and view candidate scan
-sessions collected at time-points (visits) across the study.
-
-## Searching for a candidate's scans
-
-Use the Selection Filter panel to specify search criteria such as
-candidate PSCID, DCCID, Visit Label, Site, Quality Control (QC)
-status of the scanning session, and any sequence type (T1, T2, DTI)
-in the scan session.  As well, users may filter for scans that are
-New (have not had any QC done), Pending (QC is not finalized) or both.
-
-After specifying search criteria in the selection filters, click
-"Show Data" to display results in the data table below, with one
-row per relevant scanning session. Click "Clear Form" to reset all
-selection filters.
-
-Sort the data table results by clicking on any column header.  The
-date of first acquisition of all scans for the timepoint is indicated
-under the "First Acquisition " column.  The "Last QC" column shows the
-date of the most recent Quality Control (QC) on any scan in the
-session. New scans awaiting Quality Control (QC) will be marked
-"NEW" (in red) under the "New Data" column.
-
-To download, visualize, add QC annotation or view details of imaging
-files for a candidate-timepoint, select a dataset type under the "Links"
-columns: "native", "selected" (one selected scan per modality will
-be displayed in the data query tool) or "all types".
-
-"Native" will show only raw imaging datasets, "Selected" displays
-only scans which will be displayed in the data query tool, and "all
-types" displays all data, including analyzed outputs. By clicking
-on any of these links, the user will be able to see details for
-that specific candidate's dataset from a given scan session.
+This module allows you to view candidate scan sessions collected at time-points (visits) across the study.
 
+Use the *Selection Filter* section to narrow your search. The resulting table below will automatically update with matching results as you type. 
+
+You can download, visualize, add QC annotation, or view details of imaging files for a specific candidate from one visit. To do so, click on one of the options in *Links* column. **Native** will show only raw imaging datasets. **Selected** will show only scans that will be displayed in the data query tool. **All types** displays all data, including analyzed outputs. Once you click on a link, you’ll arrive at a *View Session* page for that specific dataset.
+
+## View Session
```

Comment:
This sections looks like it should be in the help content of the view session page, not of the main menu page.

Reply:
Sounds like there's currently no way to have a separate help text for different pages within a module - I.e. one help text per module. Hopefully that can change in the future!

Reply:
if there's a file with the name of the subpage (the last part of the URL) the help is supposed to show that instead of the main module help text.. if that's not the case, it's a regression, so please file a bug report.

I was under the impression that the ones where the that couldn't be done any more were just the modules that were reactified (and as a side-effect all pages have the same URL..) In the imaging browser it's definitely still 2 distinct pages, so LORIS should load the 2 different help files for the proper page.

Reply:
thanks for clarifying @driusan. 
bug report here : https://github.com/aces/Loris/issues/5709 
(rough draft - anyone pls contribute if you want)

Reply:
@knoll-alex super-easy fix then:  un-append the View Session text, and put it back in the exact same file it used to be called. update/commit both files on this branch. 
then tag us for re-review.   we're almost there.

Reply:
If you do that, please put the Blocked tag on this PR (since merging should be Blocked by issue #5709)